### ISSUE TITLE
"duplicate declaration 'default'" if declaring commands in loaded file

### ISSUE THREAD

**Issue by BlueDrink9** [koekeishiya/skhd#95]
If I have a file loaded, `test.skhd`, that declares the default mode in order to add an opening command to it, then trying to declare a command in my original .skhdrc raises an error that prevents normal activity:

> duplicate declaration 'default'

Declaring the command in `test.skhd` works fine.

.skhdrc:
```
.load "test.skhd"
alt - h : chunkc tiling::window --focus west
```

test.skhd:
```
:: default : chunkc set focused_border_color 0xff247dcc;
alt - j : chunkc tiling::window --focus south
```

skhd version 0.3.4, OSX Mohave 10.14.3

**Comment by xaviervalarino** [comment]
I've been running into the same error. Strangely, when I moved back to my old `.skhdrc` that has modes for chunkwm, the error goes away. The error returns when I edit the old file.

**Comment by BlueDrink9** [comment]
Could you elaborate more on what you mean by that? An MWE of the difference maybe?

**Comment by xaviervalarino** [comment]
@BlueDrink9 What do you mean by MWE? In both cases my `.skhdrc` is a single file.
The only difference:
- one is for chunkwm and hasn't changed in the last six months
- The new one is so that I could migrate to yabai.

I wanted to keep the modal behavior, which led me to this issue.

**Comment by BlueDrink9** [comment]
MWE = minimum working example. Cut bits out of the two files until one works and one doesn't.

Interesting that you get the same issue with a single file. Do you declare the default mode multiple times?

**Comment by xaviervalarino** [comment]
Feel a little bit dumb about this one; I was using a colon where I should have been using a semicolon when entering a new mode (see example below). I'm guessing this looked like a declaration to skhd, but I haven't been able to get the same `duplicate declaration 'default'` error as before.
```
:: default  : yabai -m config window_border off
:: focus  @ : yabai -m config window_border on

default < hyper - space : focus
focus   < escape        : default
```

**Comment by snaveevans** [comment]
I'm getting the same error.  I've tried copying the command from the example and tried the actual logic I wish to perform `:: default : yabai -m config normal_window_border_color 0xff505050`

**Comment by xaviervalarino** [comment]
@snaveevans, did you make sure you're using a semicolon when entering and exiting modes?

**Comment by snaveevans** [comment]
@xaviervalarino yes.  However even if I don't add a transition mapping adding `:: default : yabai -m config normal_window_border_color 0xff505050` to my `.skhdrc` file generates the error.

**Comment by entropitor** [comment]
@snaveevans I had a similar error. It's fixed if you put the transition mapping as the very first command. I'm guessing that internally skhd will create a default mode as soon as you create any binding. So by explicitly creating the mode afterwards, you have a duplicate mode and thus said error

**Comment by snaveevans** [comment]
@entropitor Thank you, I've confirmed that works.

**Comment by koekeishiya** [comment]
Should be resolved in v0.3.5
